templates/deployment.yaml

```diff
@@ -106,6 +106,15 @@ spec:
         - |
           curl -s -H 'Metadata-Flavor: Google' 'http://169.254.169.254/computeMetadata/v1/instance/service-accounts/default/token' --retry 30 --retry-connrefused --retry-max-time 30 > /dev/null || exit 1
       {{- end}}
+      {{- if .MountPayloadLogging }}
+      - name: fix-payload-logger-permissions
+        image: busybox:1.33.1
+        command:
+          - "chown 1000 /var/log/fluentd-payload-logger/{{.Name}}"
+        volumeMounts:
+          - mountPath: /var/log
+            name: var-log
```

Comment:
@rubasace **var-log** is shared logs directory, so use **pod-log** instead (https://github.com/estafette/estafette-extension-gke/blob/4fdddd389be3c7fe5e63c8abb44bc4ce6280b803/templates/deployment.yaml#L559-L563).
Also its better to mount it on `/var/log/travix` like in main container: https://github.com/estafette/estafette-extension-gke/blob/4fdddd389be3c7fe5e63c8abb44bc4ce6280b803/templates/deployment.yaml#L205-L207 as `/var/log` is common unix logs directory so its better not to touch it.
And then you can run chown on `/var/log/travix`

Reply:
I cannot use `pod-log` cause then the `chown` command doesn't take effect. I don't understand exactly why but it seems like K8s doesn't properly communicate the owner changes to the host when you change permissions on the root of the mounted volume (tried this with @daanlevi  and didn't fix permissions). What I've done is using `subPath` to only mount `fluentd-payload-logger` so I only mount the parent dir of the one I want to change owner of.

Doing that I've also fixed the mounting to be inside `/var/log/travix`

Reply:
Ok, so is UID `1000` is the hard-coded value we are going to use in all container running as non-root user? Then isn't it good to update fluentd logger to change owner of this path rather doing it in init-container in each application?